lib/index.ts

```diff
@@ -28,15 +26,14 @@ class PWMetrics {
     view: false,
     expectations: false,
     output: false,
-    // @todo remove when new lighthouse version be released, because -https://github.com/GoogleChrome/lighthouse/pull/1778
```

Comment:
The flag still present but
In LH v1 `disableCpuThrottling` is set up as `true` by default, we wanted it to be false.
In LH v2 `disableCpuThrottling` is `false` by default.